src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/html-report/index.html

```diff
@@ -8,8 +8,7 @@
 
 <head>
   <meta charset="UTF-8" />
-  <link rel="stylesheet"
-    href="https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-core/11.1.0/css/fabric.min.css" />
```

Comment:
Should we additionally HTML encode the report JSON before embedding it into the HTML (for an additional layer of protection)?

Reply:
We added the CSP policy to avoid having to do this. I would suggest not doing it for now, as with CSP there is nothing accessible to the page anyway, and the HTML encoding is likely to be a source of bugs and fragility, not to mention slow performance of loading.